src/components/accounts/SetupImplicit.js

```diff
@@ -136,23 +144,27 @@ class SetupImplicit extends Component {
                 this.setState({ moonpayAvailable, moonpaySignedURL })
             }
         } catch (e) {
+            Mixpanel.track("CA Check moonpay fail", {error: e.message})
             console.warn('Error checking Moonpay', e);
         }
     }
 
     checkBalance = async () => {
+        Mixpanel.track("CA Check balance from implicit start")
         const { implicitAccountId } = this.props
 
         const account = new nearApiJs.Account(this.connection, implicitAccountId)
         try {
             const state = await account.state()
             if (new BN(state.amount).gte(MIN_BALANCE_TO_CREATE)) {
+                Mixpanel.track("CA Check balance from implicit finish")
```

Comment:
This will only fire if the implicit account balance is sufficient. Maybe say `CA Check balance from implicit: sufficient` to be more clear.

And then add an `else` with `Mixpanel.track("CA Check balance from implicit: insufficient")`